src/sbd-cluster.c

```diff
@@ -447,7 +454,7 @@ sbd_remote_check(gpointer user_data)
         }
         exe_path[rc] = 0;
 
-        rc = snprintf(expected_path, sizeof(proc_path), "%s/pacemaker_remoted", SBINDIR);
+        rc = snprintf(expected_path, sizeof(proc_path), "%s/%s", SBINDIR, PACEMAKER_REMOTE_BINARY);
```

Comment:
Since the expected path is made of all literal strings, we can avoid memory usage by using  SBINDIR "/" PACEMAKER_REMOTE_BINARY instead of expected_path

Reply:
True lazily hadn't checked what was behind SBINDIR ;-)
Shouldn't have been implemented like that in the first place.
Good time for a cleanup now.